.github/workflows/ci.yaml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: CI Test (PR)
```

Comment:
`pr-ci.yml` is the one that runs against PRs. This file is the one that runs on pushes to `main`, or when PRs are merged to `main`, so I think this rename is a bit inaccurate.

Reply:
oh yes! thanks for catching that!